backend/.claude/skills/create-issue/SKILL.md

```
@@ -0,0 +1,100 @@
+| chore | feature-template | `⚙️chore` |
+| docs | feature-template | `📝docs` |
+| test | feature-template | `🧪 test` |
+
+## 2. 현재 이슈 템플릿
+
+아래 내용을 참고해 이슈 본문을 작성한다.
+
+### Bug 템플릿
+!`cat ../.github/ISSUE_TEMPLATE/bug_report.md`
+
+### Feature 템플릿
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)